android/src/main/java/com/usabilla/reactnative/ubform/UsabillaBridge.java

```diff
@@ -128,13 +128,7 @@ public String getName() {
      */
     @ReactMethod
     public void initialize(@NonNull String appId) {
```

Comment:
On Android since JS load does not necessary start an Activity. So if we put `Usabilla.initialize` outside of Application React Component (ex: global JS scope), this fails.
At the same time, because Android Activities are sometimes killed when they are in background, having a reference to the fragment manager here is not safe.